src/main/resources/db/demo/kdt/99_verify.sql

```
@@ -206,3 +206,38 @@ FROM `text` WHERE txt_id BETWEEN 8001 AND 8144
 --    GET /api/v1/approvals?scope=pending&size=1
 --    GET /api/v1/bidding/notices
 --    GET /api/v1/finance/settlements
+
+
+-- =====================================================================
+-- KB 프로젝트(P8011) 검증 — 전부 0이어야 정상 (2026-08-17 throwaway MySQL 8 에서 통과 확인)
+-- =====================================================================
+SELECT 'KB col_span 규칙위반' k, COUNT(*) v FROM block
+  WHERE block_id BETWEEN 8300 AND 8422
+    AND ((type IN ('FILE','BID_NOTICE') AND col_span<>2) OR (type NOT IN ('FILE','BID_NOTICE') AND col_span<>1))
+UNION ALL SELECT 'KB 정산 실지급≠연결합계', COUNT(*) FROM settlement_block s WHERE s.settle_id BETWEEN 8012 AND 8019
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)